create_infotexts.py

```diff
@@ -198,37 +198,47 @@ def main():
     :metadata_json: created with script `metadata_to_json_and_fnamesmap.py
     """
     metadata_json = "SMVK-Cypern_2017-01_metadata.json"
-    outpath = "./infofiles/"
+    outfile = open("./SMVK-Cypern_2017-02_wikiformat_data.json", "w")
 
     # Hack to printout a wikitable to copy-paste to WikiCommons
     people = create_people_mapping_wikitable(people_mapping)
     # print(people + "\n")
 
     metadata = load_json_metadata(metadata_json)
+    batch_dict = {}
     for fotonr in metadata:
+        batch_dict[fotonr] = {}
```

Comment:
I would create a variable here and then only add it to the batch_dict at the end of the loop. This makes it easier to refer to (e.g. `image_info` instead of `batch_dict[fotonr]`), easier to change (if instead of `fotonr` you need to use another value you only change in one place) and it means you can skip adding it if something has gone wrong.